palindrome

```diff
@@ -0,0 +1,18 @@
+import java.io.*;
+import java.util.*;
+
+public class Solution {
+
+    public static void main(String[] args) {
+        
+        Scanner sc=new Scanner(System.in);
+        String A=sc.next();
+        String a="";
+        for(int i=A.length()-1;i>=0;i--)
+        a+=A.charAt(i);
```

Comment:
```suggestion
        String a = new StringBuilder(A).reverse().toString();
```

Less code and should have better performance. Best would be to iterate from both ends of A instead of allocating any extra strings though.